kubeapps-dashboard.jsonnet

```diff
@@ -87,6 +87,12 @@ local serviceDeployFromValues(parentName, componentName, values) = {
     },
   },
 
+  tillerServiceAccount: kube.ServiceAccount("tiller") + $.namespace,
+  tillerBinding: kube.ClusterRoleBinding("tiller-cluster-admin") {
+    roleRef_: kube.ClusterRole("cluster-admin"),
```

Comment:
The cluster-admin role is an already existing role in the cluster

Reply:
can we be sure that the `cluster-admin` role exists by design in every cluster ?

Reply:
It is one of the k8s RBAC bootstrap policies, so is created/updated by default by k8s itself. The good news for kubeapps is that we only have to care about what exists on minikube and GKE.

This seems like something better explored/enforced by a kubeapps integration test somewhere, but I manually checked default installs of GKE 1.7.8, 1.8.3; minikube 1.7.5, 1.8.0:
- GKE includes `cluster-admin` (with the standard definition).
- minikube with the default bootstrapper (localkube) *does not* have `cluster-admin` by default.  It requires `--extra-config apiserver.Authorization.Mode=RBAC` or `--bootstrapper kubeadm` in order to have this clusterrole created.

In light of the latter, I suggest we need to include our own `cluster-admin` equivalent clusterrole (not called `cluster-admin`)

Reply:
@anguslees I checked this on a default `minikube start` and it doesn't complain about the ClusterRoleBinding. I think this is because, when RBAC is disabled, it seems to just ignore all RBAC resources and fail silently. This is also why `kubeapps up` just works™ on minikube too, despite all the RBAC resources it tries to create.

Reply:
MSFT people are testing kubeapps on Azure as we speak, so this may be an issue for them. we need to keep track of it.